File: kattis/pizza_crust.py
# https://open.kattis.com/problems/pizza2
# 1s
# 1.7 Easy

# 50 chars Would be on Top 10 place 5 but my Prolog program is in place 3.
r,c=map(int,input().split())
print(100*(1-c/r)**2)


# 52 chars
# r,c=map(int,input().split())
# print(100*((r-c)/r)**2)

# 55 chars
# r,c=map(int,input().split())
# print(100*((r-c)**2/r**2))

File: kattis/pizza_crust.rb
# https://open.kattis.com/problems/pizza2
# 1s
# 1.7 Easy

# 43 chars Would be on Top 10 place 3 but I will keep my Prolog program there instead (47 chars)
# So this is not submitted...
r,c=gets.split.map(&:to_f)
p 100*(1-c/r)**2
